server/events/output_updater.go

```diff
@@ -70,6 +70,56 @@ func (c *ChecksOutputUpdater) UpdateOutput(ctx *command.Context, cmd PullCommand
 		return
 	}
 
+	// Handle ApprovePolicies command separately
+	if cmd.CommandName() == command.ApprovePolicies {
+		output := c.MarkdownRenderer.Render(res, cmd.CommandName(), ctx.Pull.BaseRepo)
+		updateStatusReq := types.UpdateStatusRequest{
+			Repo:        ctx.HeadRepo,
+			Ref:         ctx.Pull.HeadCommit,
+			StatusName:  c.TitleBuilder.Build(cmd.CommandName().String()),
+			PullNum:     ctx.Pull.Num,
+			Description: fmt.Sprintf("%s succeded", strings.Title(cmd.CommandName().String())),
+			Output:      output,
+			State:       models.SuccessCommitStatus,
+		}
+
+		if err := c.VCSClient.UpdateStatus(ctx.RequestCtx, updateStatusReq); err != nil {
+			ctx.Log.Error("updable to update check run", map[string]interface{}{
+				"error": err.Error(),
+			})
+		}
+
+		// In addition, update project level atlantis/policy_check checkruns
+		for _, projectResult := range res.ProjectResults {
+			statusName := c.TitleBuilder.Build(command.PolicyCheck.String(), vcs.StatusTitleOptions{
+				ProjectName: projectResult.ProjectName,
+			})
+
+			var state models.CommitStatus
+			if projectResult.Error != nil || projectResult.Failure != "" {
+				state = models.FailedCommitStatus
+			} else {
+				state = models.SuccessCommitStatus
+			}
+
+			updateStatusReq := types.UpdateStatusRequest{
+				Repo:       ctx.HeadRepo,
+				Ref:        ctx.Pull.HeadCommit,
+				StatusName: statusName,
+				PullNum:    ctx.Pull.Num,
+				State:      state,
+			}
+
+			if err := c.VCSClient.UpdateStatus(ctx.RequestCtx, updateStatusReq); err != nil {
+				ctx.Log.Error("updable to update check run", map[string]interface{}{
```

Comment:
```suggestion
				ctx.Log.Error("unable to update check run", map[string]interface{}{
```

Also pls use the ctx specific log functions for this and all the other PRs you have.